.scripts/summarize_progress/main.py

```diff
@@ -0,0 +1,138 @@
+import re
+import polib
+import glob
+import datetime
+import requests
+
+from pathlib import Path
+
+
+def entry_check(pofile: polib.POFile) -> str:
+    '''
+    Check the po file with how many entries are translated or not.
+    '''
+
+    lines_tranlated = len(pofile.translated_entries())
+    lines_untranlated = len(pofile.untranslated_entries())
+
+    if lines_tranlated == 0:
+        result = "❌"
+    elif lines_untranlated == 0:
+        result = "✅"
+    else:
+        lines_all = lines_tranlated + lines_untranlated
+        progress = lines_tranlated / lines_all
+        progress_percentage = round(progress * 100, 2)
+        result = f"Ongoing, {progress_percentage} %"
+
+    return result
+
+
+def get_open_issues_count() -> int:
+    '''
+    Fetch GitHub API to get the number of OPEN ISSUES.
+    '''
+
+    url = f"https://api.github.com/search/issues?q=repo:python/python-docs-zh-tw+type:issue+state:open"
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    r = requests.get(url=url, headers=headers)
+    result = r.json()
+
+    return result["total_count"]
+
+
+def get_github_issues() -> list:
+    '''
+    Fetch GitHub API to collect the infomation of OPEN ISSUES,
+    including issue title and assignee.
+
+    Steps:
+    1. Fetch GitHub API and get open issue list
+    2. Filter the issue if it have no assignee
+    3. Filter the issue if it have no "Translate" in the title
+    4. Filter the issue if it have no correct filepath in the title
+    '''
+    NUMBER_OF_ISSUES = get_open_issues_count()
+
+    url = f"https://api.github.com/search/issues?q=repo:python/python-docs-zh-tw+type:issue+state:open&per_page={NUMBER_OF_ISSUES}"
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+    r = requests.get(url=url, headers=headers)
+    result = r.json()
+
+    result_list = []
+    for issue in result["items"]:
+        if issue["assignee"] is None:
+            continue
+
+        title = issue["title"]
+        if "翻譯" not in title and "translate" not in title.lower():
+            continue
+
+        match = re.search("(?P<dirname>[^\s`][a-zA-z-]+)/(?P<filename>[a-zA-Z0-9._-]+(.po)?)", title)
+        if not match:
+            continue
```

Comment:
這邊 regex 前段我新增了 `[^\s`]`，不然會有 bug

```suggestion
        match = re.search("(?P<dirname>[^\s`][a-zA-z-]+)/(?P<filename>[a-zA-Z0-9._-]+(.po)?)", title)
        if not match:
            continue
```

Reply:
Previous would be like this

![image](https://github.com/python/python-docs-zh-tw/assets/34214497/e19cf369-38f2-46b5-9da4-15eb8292bd8b)